From a66a96e92638a71f3b054c5988d26b6a4544cd36 Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Tue, 25 Aug 2026 16:58:07 +0100
Subject: [PATCH] Remove schema handling from transpile.sh

---
 scripts/transpile.sh | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/scripts/transpile.sh b/scripts/transpile.sh
index 4995022..db2950c 100755
--- a/scripts/transpile.sh
+++ b/scripts/transpile.sh
@@ -4,7 +4,6 @@ set -e
 rm -rf _build
 
 echo Compiling into target/...
-glib-compile-schemas schemas &
 for proj in ${PROJECTS}; do
     mkdir -p _build/"${proj}"
     npx tsc --p src/"${proj}"
@@ -15,7 +14,7 @@ ls target
 echo
 
 echo Packing into _build/...
-cp -r metadata.json icons schemas ./*.css _build &
+cp -r metadata.json icons ./*.css _build &
 shopt -s globstar nullglob
 for src in target/**/*.js; do
     dst="${src//target/_build}"
-- 
2.55.0

