From a823014b012c0908c5838106366a149ee1e0b433 Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Wed, 21 Oct 2020 00:20:02 -0500
Subject: [PATCH] Open shortcuts website if pop-shell-shortcuts is not
 installed

---
 src/panel_settings.ts | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/panel_settings.ts b/src/panel_settings.ts
index a2874f1..9ee9c68 100644
--- a/src/panel_settings.ts
+++ b/src/panel_settings.ts
@@ -1,7 +1,6 @@
 const Me = imports.misc.extensionUtils.getCurrentExtension();
 
 // import * as auto_tiler from 'auto_tiler';
-import * as log from 'log';
 import * as Utils from 'utils';
 
 //import type { Entity } from './ecs';
@@ -94,7 +93,7 @@ function settings_button(menu: any): any {
         if (path) {
             imports.misc.util.spawn([path]);
         } else {
-            log.error(`You must install \`pop-shell-shortcuts\``)
+            imports.misc.util.spawn(['xdg-open', 'https://support.system76.com/articles/pop-keyboard-shortcuts/']);
         }
 
         menu.close();
@@ -141,7 +140,7 @@ function shortcuts(menu: any): any {
         if (path) {
             imports.misc.util.spawn([path]);
         } else {
-            log.error(`You must install \`pop-shell-shortcuts\``)
+            imports.misc.util.spawn(['xdg-open', 'https://support.system76.com/articles/pop-keyboard-shortcuts/']);
         }
 
         menu.close();
-- 
2.28.0

