From bf4956e65e359dcab9724efc22e950f411e7d05e Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Wed, 21 Oct 2020 00:44:51 -0500
Subject: [PATCH] Remove schemas from compile target

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index cd37b83..c5dea22 100644
--- a/Makefile
+++ b/Makefile
@@ -37,10 +37,10 @@ configure:
 convert: transpile
 	sh scripts/transpile.sh
 
-compile: convert metadata.json schemas
+compile: convert metadata.json
 	rm -rf _build
 	mkdir -p _build
-	cp -r metadata.json icons schemas target/*.js imports/*.js *.css _build
+	cp -r metadata.json icons target/*.js imports/*.js *.css _build
 	for proj in $(PROJECTS); do \
 		mkdir -p _build/$${proj}; \
 		cp -r target/$${proj}/*.js _build/$${proj}; \
-- 
2.28.0

